codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoderTest.java

```diff
@@ -183,6 +185,26 @@ public void dataReadAfterGoAwayShouldApplyFlowControl() throws Exception {
     }
 
     @Test
+    public void dataReadForUnknownStreamShouldApplyFlowControl() throws Exception {
```

Comment:
We should also test that an "invalid" stream ID should fail and throw an exception if we already don't check for this.